File: chroma-league/src/test/java/com/bonepl/chromaleague/hud/animations/AllyInfernalDragonKillAnimationTest.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.hud.AnimationTester;
import org.junit.jupiter.api.Test;

class AllyInfernalDragonKillAnimationTest {

    @SuppressWarnings("JUnitTestMethodWithNoAssertions")
    @Test
    void playAllyInfernalDragonKillAnimation() {
        new AnimationTester().testAnimation(new AllyInfernalDragonKillAnimation());
    }
}

File: test/hud/animations/AllyInfernalDragonKillAnimation.test.ts
import {describe, test} from 'vitest';
import {AnimationTester} from '../../AnimationTester.js';
import {AllyInfernalDragonKillAnimation} from '../../../src/hud/animations/AllyInfernalDragonKillAnimation.js';

/**
 * Integration test for ally infernal dragon kill animation on actual hardware.
 */
describe('AllyInfernalDragonKillAnimation Integration', () => {
    test('playAllyInfernalDragonKillAnimation', async () => {
        await new AnimationTester().testAnimation(new AllyInfernalDragonKillAnimation());
    }, 30000);
});
